.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  env: {
+    es2020: true,
+  },
+  extends: [
+    'plugin:@typescript-eslint/recommended'
```

Comment:
add airbnb config. Your linter skips anused variables. ts recommended is not enough.